cilium-dbg/cmd/status.go

```diff
@@ -50,6 +51,7 @@ func init() {
 	statusCmd.Flags().BoolVar(&statusDetails.AllClusters, "all-clusters", false, "Show all clusters")
 	statusCmd.Flags().BoolVar(&allHealth, "all-health", false, "Show all health status, not just failing")
 	statusCmd.Flags().BoolVar(&brief, "brief", false, "Only print a one-line status message")
+	statusCmd.Flags().BoolVar(&withoutK8sConnectivity, "without-k8s-connectivity", false, "whether the result of connectivity check to kube-apiserver should affect the result of health status of daemon")
```

Comment:
Do you mind syncing the description here with the one used in `openapi.yaml`? They seem to be different.

Also, slight wording suggestion:

```suggestion
	statusCmd.Flags().BoolVar(&withoutK8sConnectivity, "without-k8s-connectivity", false, "Controls if the result of the connectivity check to the kube-apiserver impacts the health status of the daemon")
```